sunflower-land/sunflower-land
# [FIX] Box overlay position

# Description

- fix box overlay displaying outside box
- attempt to standardize pixel sizes for images inside box

**Overlay**
Before  |  After
---  |  ---
![image](https://user-images.githubusercontent.com/107602352/201978769-69a6a318-23d3-4ffd-a989-646ac537ba77.png)  |  ![image](https://user-images.githubusercontent.com/107602352/201978797-c89f4909-3f79-4f22-a68e-69fb48723057.png)

**Pixel sizes**
Before  |  After
---  |  ---
![image](https://user-images.githubusercontent.com/107602352/201979130-2924b950-167f-4503-b2ad-061e322309d5.png)  |  ![image](https://user-images.githubusercontent.com/107602352/201979154-332e8a8e-af14-4e45-8737-722348b5c33a.png)

## Type of change


# How Has This Been Tested?

- open inventory
- check the boxes that are locked/pending

# Checklist:
